src/server/svr_mail.c

```diff
@@ -180,6 +181,9 @@ svr_mailowner_id(char *jid, job *pjob, int mailpoint, int force, char *text)
 	/* Unprotect child from being killed by kernel */
 	daemon_protect(0, PBS_DAEMON_PROTECT_OFF);
 
+	if ((mailer = server.sv_attr[(int)SVR_ATR_mailer].at_val.at_str) == 0)
```

Comment:
Perhaps you want to use the new function is_attr_set() along with checking the value is NULL (and using the NULL word explicitly instead of 0 is nicer to read)

Reply:
I used is_attr_set() and tested for true. I did the same for _SVR_ATR_mailfrom_.